src/style.css

```diff
@@ -0,0 +1,93 @@
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.Snake_box {
+  font-family: Arial, Helvetica, sans-serif;
+  display: flex;
+  align-items: center;
+  height: 300px;
+  flex: 0 0 33.33%;
+  justify-content: center;
+  color: white;
+  font-size: 100px;
+}
+
+.Snake_box:nth-child(1) {
+  background-color: #ff0102;
+}
+
+.Snake_box:nth-child(2) {
+  background-color: #cc0202;
+}
+
+.Snake_box:nth-child(3) {
+  background-color: #9a0002;
+}
+
+.Snake_box:nth-child(4) {
+  background-color: #670100;
+}
+
+.Snake_box:nth-child(5) {
+  background-color: #340000;
+}
+
+.Snake_box:nth-child(6) {
+  background-color: #000;
+}
+
+@media (min-width: 900px) {
+  .Snake_box:nth-child(4) {
+    order: 3;
+  }
+  .Snake_box:nth-child(5) {
+    order: 2;
+  }
+}
+
+@media (max-width: 899px) {
+  .Snake_box {
+    flex: 0 0 50%;
```

Comment:
you no needed to repeat `flex-shrink` and `flex-grow` here
use only `flex-basis`